services/sources.service.ts

```
@@ -0,0 +1,45 @@
+    const response = await apiService.apiUrl.post(
+      `https://be.crowdy.chat/api/v1/sources/facebook/${req.pageId}/leads`,
+      {},
+      {
+        headers: {
+          "X-Organization-ID": req.organizationId,
+          Authorization: `Bearer ${req.firebaseId}`,
+        },
+      }
+    );
+  };
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)